.gitignore

```diff
@@ -22,3 +22,6 @@ venv/
 
 # Build docs
 /src/array_api_typing/_version.py
+
+# Mac files
+.DS_Store
```

Comment:
To pick a nit:

This is less specific than `/src/array_api_typing/_version.py`, so it could be a bit cleaner to move this up a bit.